### ISSUE TITLE
UX [UX] System Updates - incorrect button label when an update has been selected

### ISSUE THREAD

**Issue by albanycomputers** [backdrop-contrib/eu_cookie_compliance#13]
When updating modules via admin/config/system/updates the button text does not change to "Download selected updates"

![image](https://user-images.githubusercontent.com/1705388/132917025-0d47f1e6-a9d5-4301-a510-fd851c571f25.png)

**Comment by olafgrabienski** [comment]
@albanycomputers This was recently confirmed in the Backdrop Forum, including an interesting observation in this comment: https://forum.backdropcms.org/forum/system-updates-button-name#comment-5685

**Comment by albanycomputers** [comment]
@olafgrabienski I did have a look at the issue back in 2021, but with the issue only being cosmetic, and that I'm not that conversant with the inner workings of cookies... I didn't do any more research.

**Comment by swampopus** [comment]
I can't reproduce this issue.  I'm testing on Backdrop 1.32.1.  Perhaps some underlying issue in core has been changed sufficiently that this no longer occurs?

**Comment by olafgrabienski** [comment]
Interesting! Just to be sure, the issue is supposed to happen only if EU Cookie Compliance is installed and enabled. And only in a certain scenario, as the Forum post below explains. For reference:

- https://github.com/backdrop/backdrop-issues/issues/5165#issuecomment-917193796
- https://forum.backdropcms.org/forum/system-updates-button-name#comment-5685

**Comment by swampopus** [comment]
Yes-- I have EU Cookie Compliance installed and enabled (I recently became a maintainer for this module), and I was trying to tackle some of these old issues, but I haven't been able to reproduce this one.  I definitely have the cookie being set (confirmed by looking at $_COOKIE), and the module update screen appears to work fine for me.  The button to "download selected updates" works normally.

I did recently push various bug fixes (including one that was causing a fatal error in PHP 8).  Maybe fixing that bug corrected a javascript error that was preventing the core "download selected updates" button from working?

If someone else could try to reproduce this, I'd appreciate it.  I'd like to close the issue if it isn't relevant any longer.

**Comment by olafgrabienski** [comment]
> If someone else could try to reproduce this, I'd appreciate it.

Sure! I tried it, and I wasn't able to reproduce the issue on a test site.

**Comment by swampopus** [comment]
Hmm.  I feel like it's safe for me to go ahead and close this issue.  I'm going to go ahead and do that.  Future Backdroppers-- feel free to re-open if this issues comes back up.
